server/src/.mf.config.SAMPLE.json

```diff
@@ -9,7 +9,8 @@
             "name": "core",
             "cliFlag": null,
             "cliShortFlag": null,
-            "url": "https://meta.rdkcentral.com/firebolt/api/",
+            "theBelowUrl": "Can be changed to test different versions of the firebolt-open-rpc",
```

Comment:
nit: maybe use something like "__url_Note__" or something like that. (I'm going to approve the PR anyway, but something like this will make it that much more obvious that they key is bogus.)

Reply:
Pushed change to make key url_Note